src/box/lua/slab.cc

```diff
@@ -134,38 +135,35 @@ lbox_slab_stats(struct lua_State *L)
 static int
 lbox_slab_info(struct lua_State *L)
 {
-	struct memtx_engine *memtx;
-	memtx = (struct memtx_engine *)engine_by_name("memtx");
-
-	struct allocator_stats stats;
-	memset(&stats, 0, sizeof(stats));
+	uint64_t items_size = box_slab_info(BOX_SLAB_INFO_ITEMS_SIZE);
+	uint64_t items_used = box_slab_info(BOX_SLAB_INFO_ITEMS_USED);
+	uint64_t arena_size = box_slab_info(BOX_SLAB_INFO_ARENA_SIZE);
+	uint64_t arena_used = box_slab_info(BOX_SLAB_INFO_ARENA_USED);
+	uint64_t quota_size = box_slab_info(BOX_SLAB_INFO_QUOTA_SIZE);
+	uint64_t quota_used = box_slab_info(BOX_SLAB_INFO_QUOTA_USED);
 
 	/*
 	 * List all slabs used for tuples and slabs used for
 	 * indexes, with their stats.
 	 */
 	lua_newtable(L);
-	allocators_stats(&stats);
-	struct mempool_stats index_stats;
-	mempool_stats(&memtx->index_extent_pool, &index_stats);
-
-	double ratio;
 	char ratio_buf[32];
+	double ratio = 100 * ((double)(items_used)
+			/ ((double)(items_size) + 0.0001));
```

Comment:
Please don't split lines if they fit in 80 characters. Please remove extra parentheses and conversions.

```diff
diff --git a/src/box/lua/slab.cc b/src/box/lua/slab.cc
index 2b62813be162..7a197fb420ce 100644
--- a/src/box/lua/slab.cc
+++ b/src/box/lua/slab.cc
@@ -148,11 +148,8 @@ lbox_slab_info(struct lua_State *L)
 	 */
 	lua_newtable(L);
 	char ratio_buf[32];
-	double ratio = 100 * ((double)(items_used)
-			/ ((double)(items_size) + 0.0001));
-
-	snprintf(ratio_buf, sizeof(ratio_buf),
-		 "%0.2lf%%", ratio);
+	double ratio = 100 * ((double)items_used / (items_size + 0.0001));
+	snprintf(ratio_buf, sizeof(ratio_buf), "%0.2lf%%", ratio);
 
 	/** How much address space has been already touched */
 	lua_pushstring(L, "items_size");
@@ -199,11 +196,8 @@ lbox_slab_info(struct lua_State *L)
 	luaL_pushuint64(L, arena_used);
 	lua_settable(L, -3);
 
-	ratio = 100 * ((double)(arena_used) /
-		(double)(arena_size + 1));
-
-	snprintf(ratio_buf, sizeof(ratio_buf),
-		 "%0.1lf%%", ratio);
+	ratio = 100 * ((double)arena_used / (arena_size + 1));
+	snprintf(ratio_buf, sizeof(ratio_buf), "%0.1lf%%", ratio);
 
 	lua_pushstring(L, "arena_used_ratio");
 	lua_pushstring(L, ratio_buf);
@@ -231,11 +225,8 @@ lbox_slab_info(struct lua_State *L)
 	 * factor, it's the quota that give you OOM error in the
 	 * end of the day.
 	 */
-	ratio = 100 * ((double)(quota_used) /
-		(double)(quota_size + 0.0001));
-
-	snprintf(ratio_buf, sizeof(ratio_buf),
-		 "%0.2lf%%", ratio);
+	ratio = 100 * ((double)quota_used / (quota_size + 0.0001));
+	snprintf(ratio_buf, sizeof(ratio_buf), "%0.2lf%%", ratio);
 
 	lua_pushstring(L, "quota_used_ratio");
 	lua_pushstring(L, ratio_buf);
```

Reply:
Fix it